src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random random = new Random();
+    private static final int FIGURES_COUNT = 5;
+    private static final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        final int randomSizeRange = random.nextInt(10) + 1;
```

Comment:
If we consider `10` as a max value in range, it could also be stored as constant.
Also I'm not sure if 'range' in variable name is correct - from what I see you picked a random fixed value here so it's not range anymore.